src/app/collection/shared/included/curators/curator-card/curator-card.component.html

```diff
@@ -1,5 +1,30 @@
-<div>
-    <img class = "grav" *ngIf="profileImg" src="{{ profileImg }}"/>
-    <p>{{curator.name|titlecase}}</p>
-    <p>{{organizationFormat(curator.organization)}}</p>
-</div>
+<div class="curator-card">
+    <div [ngClass] = "{
+        'card-content': !mobile,
+        'card-content-mobile': mobile
+    }">
+        <div class="icon">
+        <img class="grav" *ngIf="profileImg" [src]="profileImg" alt="Curator's Logo" />
+        </div>
+        <div class="name">
+        <p>{{ curator.name | titlecase }}</p>
+        </div>
+        <div class="organization">
+        <p>{{ organizationFormat(curator.organization) }}</p>
+        </div>
+    </div>
+</div>  
+<!-- <div class="curator-card">
+    <div class="card-content">
+      <div class="item">
+        <img class="grav" *ngIf="profileImg" [src]="profileImg" alt="Curator's Logo" />
+      </div>
+      <div class="item name">
+        <p>{{ curator.name | titlecase }}</p>
+      </div>
+      <div class="item organization">
+        <p>{{ organizationFormat(curator.organization) }}</p>
+      </div>
+    </div>
+  </div> -->
```

Comment:
remove(?)

```suggestion
```

Reply:
Yup sorry forgot to delete, just got removed